.github/workflows/linters.yml

```diff
@@ -0,0 +1,36 @@
+name: Linters
```

Comment:
### Optional
- [ ] It would be nice if we improve on our PR by adding a little bit of information about the project, what we did. Here is the [project description](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md) where we can find points to add